sponsors/migrations/0086_auto_20220809_1655.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 2.2.24 on 2022-08-09 16:55
```

Comment:
exclude this file for now please, I'll add a followup PR to include this dangling migration.

Reply:
Moved this to https://github.com/python/pythondotorg/pull/2114.